guide/02-api-overview/release-notes.ipynb

```diff
@@ -1,5 +1,517 @@
 {
```

Comment:
<ul><li>Adds parameter tables to documentation</li></ul><p>Can you convert this line to -</p><p>Enhanced the documentation with more details and converted the documentation to tabular format. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1101/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>Done!</p>